lib/AST/ASTDumper.cpp

```diff
@@ -609,7 +609,12 @@ namespace {
     void visitExtensionDecl(ExtensionDecl *ED) {
       printCommon(ED, "extension_decl", ExtensionColor);
       OS << ' ';
-      ED->getExtendedType().print(OS);
+      if (auto nominalType = ED->getExtendedType()->getAnyNominal()) {
+        nominalType->getDeclaredInterfaceType().print(OS);
+      }
+      else {
```

Comment:
Stylistic nit: `else` should be following the closing brace (we use the same formatting scheme as [LLVM](https://llvm.org/docs/CodingStandards.html))